src/test/groovy/com/vacation_bot/vacation/VacationServiceUnitTest.groovy

```diff
@@ -0,0 +1,46 @@
+package com.vacation_bot.vacation
+
+import com.vacation_bot.AbstractSpockUnitTest
+import com.vacation_bot.core.vacation.VacationService
+import com.vacation_bot.domain.models.UserModel
+import com.vacation_bot.domain.models.VacationTotal
+import com.vacation_bot.repositories.UserModelRepository
+import com.vacation_bot.repositories.VacationModelRepository
+import com.vacation_bot.repositories.VacationTotalRepository
+
+
+class VacationServiceUnitTest extends AbstractSpockUnitTest{
+
+    def 'exercise createVacation'() {
+        given: 'given valid subject under test'
+        def userModelRepository = Mock(UserModelRepository)
+        def vacationTotalRepository = Mock(VacationTotalRepository)
+        def vacationModelRepository = Mock(VacationModelRepository)
+
+        def vacationService = new VacationService(vacationModelRepository,
+                vacationTotalRepository, userModelRepository)
+
+        and: 'valid input data'
+        def userName = 'Alex'
+        def startDate = '2017-10-02'
+        def endDate = '2017-10-20'
+
+        when: 'the exercised method is called'
+        def result = vacationService.createVacation(userName, startDate, endDate)
+
+        then: 'result'
```

Comment:
The better comment will be `the repositories are called as expected` or `repositories return expected values` or something like this.